<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <class>QG_CommandWidget</class>
  <widget class="QWidget" name="QG_CommandWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>102</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Command Line</string>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>
      <item>
        <widget class="QTextEdit" name="teHistory">
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>23</height>
            </size>
          </property>
          <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="lineWidth">
            <number>0</number>
          </property>
          <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="linkUnderline" stdset="0">
            <bool>false</bool>
          </property>
          <property name="documentTitle">
            <string notr="true"/>
          </property>
          <property name="readOnly">
            <bool>true</bool>
          </property>
          <property name="undoRedoEnabled">
            <bool>false</bool>
          </property>
          <property name="html">
            <string notr="true"></string>
          </property>
          <property name="acceptRichText">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="line1">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QLabel" name="lCommand">
              <property name="text">
                <string>Command:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QG_CommandEdit" name="leCommand">
              <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>QG_CommandEdit</class>
      <extends>QWidget</extends>
      <header location="local">qg_commandedit.h</header>
      <sizehint>
        <width>-1</width>
        <height>12</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections>
    <connection>
      <sender>leCommand</sender>
      <signal>returnPressed()</signal>
      <receiver>QG_CommandWidget</receiver>
      <slot>trigger()</slot>
    </connection>
    <connection>
      <sender>leCommand</sender>
      <signal>tabPressed()</signal>
      <receiver>QG_CommandWidget</receiver>
      <slot>tabPressed()</slot>
    </connection>
    <connection>
      <sender>leCommand</sender>
      <signal>escape()</signal>
      <receiver>QG_CommandWidget</receiver>
      <slot>escape()</slot>
    </connection>
    <connection>
      <sender>leCommand</sender>
      <signal>focusIn()</signal>
      <receiver>QG_CommandWidget</receiver>
      <slot>setCommandMode()</slot>
    </connection>
    <connection>
      <sender>leCommand</sender>
      <signal>focusOut()</signal>
      <receiver>QG_CommandWidget</receiver>
      <slot>setNormalMode()</slot>
    </connection>
  </connections>
</ui>
